### ISSUE TITLE
System Dashboard edit links accessibility

### ISSUE THREAD

**Issue by brianteeman** [joomla/backend-template#242]
All the edit links present for each "block" on the dashboard have the exact same name which means that it is not accessible.

In addition the icon should be aria-hidden for consistent behaviour with the rest of Joomla

![image](https://user-images.githubusercontent.com/1296369/58381017-a7575180-7fb0-11e9-93f8-fe9d1f89fb5a.png)

The same is true for each component submenu
![image](https://user-images.githubusercontent.com/1296369/58381048-ea192980-7fb0-11e9-9e2d-81aac87d89a5.png)

**Comment by chmst** [comment]
Closing as resolved
